tests/test_api.py

```diff
@@ -1225,6 +1232,12 @@ def test_get_nodes(self, api_test_project):
             assert n[0] == api_test_project.nodes[index].name
             assert n[1] == api_test_project.nodes[index].node_type
 
+    def test_arrange_nodes(self, api_test_project):
```

Comment:
Remember to also change the name of the test function

Reply:
Done